apm-agent-plugins/apm-scala-concurrent-plugin/pom.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>apm-agent-plugins</artifactId>
+        <groupId>co.elastic.apm</groupId>
+        <version>1.15.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>apm-scala-concurrent-plugin</artifactId>
+    <name>${project.groupId}:${project.artifactId}</name>
+
+    <properties>
+        <apm-agent-parent.base.dir>${project.basedir}/../..</apm-agent-parent.base.dir>
+        <maven.compiler.target>9</maven.compiler.target>
+        <maven.compiler.testTarget>9</maven.compiler.testTarget>
+    </properties>
+
+    <dependencies>
+<!--        <dependency>-->
+<!--            <groupId>${project.groupId}</groupId>-->
+<!--            <artifactId>apm-java-concurrent-plugin</artifactId>-->
+<!--            <version>${project.version}</version>-->
+<!--            <scope>test</scope>-->
+<!--        </dependency>-->
+<!--        <dependency>-->
+<!--            <groupId>${project.groupId}</groupId>-->
+<!--            <artifactId>apm-java-concurrent-plugin</artifactId>-->
+<!--            <version>${project.version}</version>-->
+<!--            <type>test-jar</type>-->
+<!--            <scope>test</scope>-->
+<!--        </dependency>-->
```

Comment:
Currently, disabling this dependency since adding it not only brings in the Wrapper classes, but for some reason also pulls in the JavaInstrumentation and method-matchers from the java-concurrent-plugin.
Comments welcome on how to prevent that :)

Reply:
Shouldn't the unit test only test the Scala instrumentation? If now the JavaConcurrent instrumentation also starts running, it makes it more difficult to for example check the debug logs of the tests since it gets 'polluted' with the JavaConcurrent instrumentation.

Reply:
I was under the impression that you were trying to leverage the executor instrumentation from the java concurrent plugin?

Reply:
Still unsure about it to be honest.
My question was mainly pointed towards you and the expectations around how plugins get tested and if there are any restrictions on how to test it properly.